.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Allow GitHub to request an OIDC JWT ID token, for exchange with `aws-actions/configure-aws-credentials`
+      # See https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#updating-your-github-actions-workflow
+      id-token: write
+
+      # Required for `actions/checkout`
+      contents: read
+
+    steps:
+      # Checkout the branch
+      - uses: actions/checkout@v3
+
+      # Java is needed for the Scala Play app
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '8'
```

Comment:
This project [currently requires Java 8](https://github.com/guardian/login.gutools/blob/54c7067f71b98c689f174d704c018e0135bf6394/build.sbt#L67).

This value should get updated when moving to Java 11, e.g. #61.